spec/requests/api/document_create_request_put_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe "API request to patch document create requests", type: :request, show_exceptions: true do
+  include ActionDispatch::TestProcess::FixtureFile
+
+  let!(:api_user) { create :api_user }
+  let!(:planning_application) { create(:planning_application, local_authority: @default_local_authority) }
+  let!(:document) { create(:document, :with_file, :public, planning_application: planning_application, validated: false, invalidated_document_reason: "Not readable") }
```

Comment:
This document isn't needed for this test. You've copied it over from the replace document spec, but we're not replacing a document here :-)